builtin/game/statbars.lua

```diff
@@ -31,9 +31,8 @@ local function scaleToDefault(player, field)
 	-- Scale "hp" or "breath" to the default dimensions
 	local current = player["get_" .. field](player)
 	local nominal = core["PLAYER_MAX_" .. field:upper() .. "_DEFAULT"]
```

Comment:
Shouldn't this scale to whatever is in `item` of the respective definition? Remember that the definitions are changeable by API call.

Reply:
Sounds reasonable.